iss.py

```diff
@@ -1,11 +1,101 @@
 #!/usr/bin/env python
 
-__author__ = '???'
+__author__ = 'katran009 and google'
 
+import json
+import urllib.request
+import turtle
+import time
 
-def main():
-    pass
+# API addresses
+ASTRONAUTS_URL = 'http://api.open-notify.org/astros.json'
+LOCATION_URL = 'http://api.open-notify.org/iss-now.json'
```

Comment:
These can be factored into a single BASE_URL i.e. http://api.open-notify.org.  Then you can concatenate additional paths within GET request to build the full url/